lib/rubycritic/core/rating.rb

```diff
@@ -25,6 +25,17 @@ def to_h
     def to_json(*a)
       to_h.to_json(*a)
     end
+
+    def to_gpa
+      if @letter == "A" then 4.0
+      elsif @letter == "B" then 3.0
+      elsif @letter == "C" then 2.0
+      elsif @letter == "D" then 1.0
+      elsif @letter == "F" then 0.0
```

Comment:
Let's not mix up code with data.
How about putting the association of character to number into a hash?

```Ruby
{ 'A' => 4.0, ....}
````
Then you could just use Hash#fetch on that.